src/components/AccountContainer.js

```diff
@@ -1,64 +1,68 @@
 import React, { Component } from 'react'
 import TransactionsList from './TransactionsList'
 import Search from './Search'
+import { BankAdaptor } from '../adaptors/index'
```

Comment:
if you don't specify a file, you'll automatically pull from the `index` file in the directory